JS-Algorithms/challenges.js

```diff
@@ -14,6 +15,29 @@ Invoking "readableTime(3690)" should return "01:01:30" (HH:MM:SS)
 
 const readableTime = (seconds) => {
 	// YOUR CODE HERE...
+	let hours;
+	if (seconds >= 3600) {
+		hours = Math.floor(seconds / 3600);
+	} else {
+		hours = '00';
+	}
```

Comment:
This way is completely fine but you might make it a bit smaller with a ternary. (Although, depending on who you ask this might be less readable).

```javascript
const hours = seconds >= 3600 ? Math.floor(seconds / 3600) : '00'
```

Also, this is no issue here but when we get to typescript it might be better to make sure you're assigning the same type to a variable. e.g. using `String(Math.floor(seconds / 3600));`

Reply:
the same type of variable is something I didn't consider. Thank you.
I think the ternary is readable and I will use it to practice.